source/administration/bulk-export-data.rst

```diff
@@ -0,0 +1,320 @@
+Bulk export data
+=================
+At this time, the export supports attributes of the objects listed below.   Configuration for exporting specific areas of the server, exporting additional types of posts, reactions, file attachments, webhooks and bot messages are not yet supported. Deleted objects are not yet supported as well.  For requests to add additional attributes or objects to our exporter, please add a feature request on our `feature idea formum <https://mattermost.uservoice.com/forums/306457-general>`_.  
+
+Version object
+--------------
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">type</td>
+      <td valign="middle">string</td>
+      <td>Must be the string "version"</td>
```

Comment:
Non-blocking comment, but this reads a bit weird given it's being written by the exporter, not instructing a human how to construct this file.

Reply:
Updated (missed re-formatting in my copy/paste)